### PR TITLE
testsuite: add rust generator test

### PR DESCRIPTION
**Pull request by Riolku**
This would have caught a failure on #1148.

### CODE REVIEW

File: testsuite/rust_generator/tests/AC/test.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+language: PY3
+time: 2
+memory: 65536
+source: sol.py
+expect: [AC, AC, AC]
```
Comment (kiritofeng) [inline testsuite/rust_generator/tests/AC/test.yml:5]:
```suggestion
expect: AC
```

**Review by kiritofeng — COMMENTED** [review]
Otherwise LGTM, thanks!

File: testsuite/rust_generator/tests/AC/test.yml (hunk 1 above)
Comment (Riolku) [reply testsuite/rust_generator/tests/AC/test.yml]: Also updated the test I copied this from, lol.

**Comment by Riolku** [comment]
Also, I did in fact test this against #1148.

**Review by kiritofeng — APPROVED** [review]
LGTM, thanks!

**Pull request merged**